appserver/admingui/payara-console-extras/src/main/resources/fish/payara/admingui/extras/Strings.properties

```diff
@@ -109,6 +109,10 @@ asadminRecorder.configuration.outputLocation=Output Location
 asadminRecorder.configuration.outputLocationHelp=The absolute file to write the recorded asadmin commands to.
 asadminRecorder.configuration.filteredCommands=Filtered Commands
 asadminRecorder.configuration.filteredCommandsHelp=A comma separated list of asadmin commands to not write to file. Accepts regular expressions.
+asadminRecorder.configuration.prependOptions=Prepend Options
```

Comment:
A better name here is Prepend Enabled rather than Prepend Options 
as the combination with PrependedOptions is confusing